src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/NeoElbow/NeoElbowConstants.java

```diff
@@ -0,0 +1,42 @@
+package edu.greenblitz.robotName.subsystems.arm.elbow.NeoElbow;
+
+import com.revrobotics.CANSparkMax;
+import edu.greenblitz.robotName.RobotConstants;
+import edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants;
+import edu.greenblitz.robotName.utils.PIDObject;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+
+
+public class NeoElbowConstants {
+
+    public static final int NEO_MOTOR_ID = 1;
```

Comment:
Same comment as @NoamRosenberg08 left in `FalconElbowConstants`, no need for `NEO_` if we are inside the `NeoElbowConstants`